beacon-chain/slasher/service.go

```diff
@@ -144,6 +144,18 @@ func (s *Service) Stop() error {
 	if s.pruningSlotTicker != nil {
 		s.pruningSlotTicker.Done()
 	}
+	// Flush the latest epoch written map to disk.
+	start := time.Now()
+	log.Info("Flushing last epoch written for each validator to disk")
```

Comment:
Debug?

Reply:
I want to keep this one because I want users to wait and not forcefully shut down their machine